### DISCUSSION TITLE
Website Content

### DISCUSSION THREAD

**Discussion by kd112** [geogeeks-au/geogeeks-au.github.io#14] [General]
Starting a discussion on what kind of content we would want for the public website?

I have got some basic scaffolding setup for the website using Jekyll on this [branch](https://github.com/geogeeks-au/geogeeks-au.github.io/tree/geogeek-au.github.io-v2).
It is still using the old bootstrap template, and content (since I couldn't really think of something better). 

If we can brainstorm some content that we would want to see now, or in the future it would give some direction for the code.

**Comment by samwilson** [comment]
Great!

How do we build the new site? (sorry, I've not used Jekyll much and I think it was just `npm run build` or something)

Should we open a pull request to the master branch?

**Reply by kd112** [reply]
There is a readme on the branch with instruction on how to get it running

**Reply by kd112** [reply]
Don't think it's ready for a PR into master. 
1) It is still a WIP,
2) Its still essentially the same site

**Comment by johnwbryant** [comment]
I'm thinking we should trim it back to 4 basic sections:
- What is geogeeks (a very basic "what you need to know")
- Upcoming events (something we can hide/show easily, so that we can advertise upcoming events and turn it off if we don't have specific plans ready)
- Connect (email, slack invite link, github, plus the mailing list form)
- Thanks (Spacecubed and/or any other contributors)

Once we get the sections set up, I can try my hand at updating the content?

**Reply by kd112** [reply]
###### Just moving this here from Slack

2. website
@KD how are you going with the website, are you able to progress, or blocked? if we get it stripped back to the sections we need, i can have a go at content. not sure how possible it is, but if we get it done by next week we can start promoting a January event. (edited) 




6 replies

KD  1 hour ago
Hey @jbryant,
Been thinking about this the last couple of days,
For now,
it might be worthwhile just to rebuild the old site using Jekyll (I have already implemented most of it)
This would build the basic scaffolding for adding new content in the future, like the events  (stuff like Grant's talk on the QGIS Plugins, GeoBeers)
Also gives us time to collate the feedback we get from the survey

KD  1 hour ago
What do you think?

John Bryant  6 minutes ago
That sounds good. I don't think we need all the content though, it would work better if it was more concise. If you set up the building blocks I can help work on them. I think we can drop the hack nights and projects sections for now, but the get in touch and thanks sections are important...
:+1:
1


John Bryant  3 minutes ago
we can also talk specifics in https://github.com/geogeeks-au/geogeeks-au.github.io/discussions/14...
GitHubGitHub
Website Content · Discussion #14 · geogeeks-au/geogeeks-au.github.io
The Geogeeks public website. Contribute to geogeeks-au/geogeeks-au.github.io development by creating an account on GitHub.
:+1:
1


KD  < 1 minute ago
Works for me,
I will get rid of the hack nights and the projects  sections from the setup,
That would just keep the Masthead, MailiingList and as you said the get in touch

John Bryant  < 1 minute ago
awesome!

**Reply by johnwbryant** [reply]
@kd112 can we:
- move the mailing list form to the `contact` section
- remove the background laptop image (makes it hard to read)
